src/shared/components/Routing/routes.tsx

```diff
@@ -1,17 +1,41 @@
 import { RoutesEnum } from "../../types/Routes.enum";
 import { Route } from "../../types/Route";
 import { DetailedView } from "../../../views/DetailedView/DetailedView";
-import { TilesView } from "../../../views/TilesView/TilesView";
+import { VenuesList } from "../../../views/VenuesList";
+import { AboutUs } from "../../../views/AboutUs";
+import { Favourites } from "../../../views/Favourites";
+import { StartHosting } from "../../../views/StartHosting";
+import { LoginView } from "../../../views/Login";
 
 export const routes: Array<Route> = [
   {
     path: RoutesEnum.TilesView,
-    Component: <TilesView />,
-    name: "tiles",
+    Component: <VenuesList />,
+    name: "venues",
   },
   {
     path: RoutesEnum.DetailedView,
```

Comment:
Don't use `/venueDetails`/. You need `/venues/:id` instead with the id of a particular venue in the URL as a path parameter